### PR TITLE
Jorge/team/ehancement/updating routes

### PR DESCRIPTION
**Pull request by Jorge1772**
added route to for avatar emu, changes to the journal date

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-require('./db/config');
+require('./db/models');
```
Comment (ErnieAtLYD) [inline server/app.js:1]: hey @Jorge1772 - so help me understand - why did you rename `/db/config/index.js` to `/db/models/index.js` and rename this appropriately? I'm having a tough time wrapping my mind around it.

Comment (Jorge1772) [reply server/app.js:1]: forsome reason our connection to mongoose got moved to the models instead of the config. im not sure why that happend but looking at my code on vsc thats what im seeing, however its not this change i wanted approved its the top one. would i have to resubmit?

Comment (ErnieAtLYD) [reply server/app.js:1]: @Jorge1772 yeah. you can make your changes to this PR *OR* close this PR without merging and submit a brand new PR, your call.

Comment (Jorge1772) [reply server/app.js:1]: im not sure how that happened, is there a way to rm the change only, while still keeping the top one?

Comment (ErnieAtLYD) [reply server/app.js:1]: @Jorge1772 the sanest way to do is by adding a new commit that "reverses" the changes you previously made if that makes any sense at all.

Comment (Jorge1772) [reply server/app.js:1]: yes it does just moving the files back the original position correct?

Comment (Jorge1772) [reply server/app.js:1]: im just going to delete this file would that work?
